crates/uv-cli/src/lib.rs

```diff
@@ -1898,6 +1898,10 @@ pub struct RunArgs {
     #[arg(long, value_parser = parse_maybe_file_path)]
     pub with_requirements: Vec<Maybe<PathBuf>>,
 
+    /// Whether to show resolver and installer output from any environment modifications.
+    #[arg(long)]
+    pub show_environment: bool,
```

Comment:
Would love better ideas for this.

Reply:
This feels like a case for `-v`

Reply:
Yeah I think just `-v` is okay? I guess `-v` includes more output though? Maybe it'll be better once we do https://github.com/astral-sh/uv/issues/1569?

Reply:
I guess this is bad for the test cases though?

Reply:
This is implied by `-v`, but I thought users might want it separate from the verbose output?